### PR TITLE
[HOPSWORKS-2763] Opensearch 1.3.1

### PR DESCRIPTION
**Pull request by jimdowling**
https://logicalclocks.atlassian.net/browse/HOPSWORKS-2763

### CODE REVIEW

File: Berksfile [hunk 1]
```diff
@@ -4,24 +4,24 @@ source 'https://supermarket.chef.io'
 metadata
 
 cookbook 'conda', github: "logicalclocks/conda-chef", branch: "master"
-cookbook 'kagent', github: "logicalclocks/kagent-chef", branch: "master"
+cookbook 'kagent', github: "logicalclocks/kagent-chef", branch: "opensearch"
```
Comment (o-alex) [inline Berksfile:7]: can you please reset this to master branch

File: files/default/sql/ddl/updates/undo/2.4.0__undo.sql [hunk 2]
```diff
@@ -74,3 +74,5 @@ ALTER TABLE `hopsworks`.`jupyter_settings` DROP COLUMN `no_limit`;
 ALTER TABLE `hopsworks`.`oauth_login_state` MODIFY COLUMN `state` VARCHAR(128);
 
 ALTER TABLE `hopsworks`.`feature_group` DROP COLUMN `event_time`;
+
+DROP VIEW project_username;
```
Comment (o-alex) [inline files/default/sql/ddl/updates/undo/2.4.0__undo.sql:78]: is this needed? if yes can we move it to 2.6.0

File: recipes/install.rb [hunk 3]
```diff
@@ -890,3 +890,13 @@
   mode "770"
   action :create
 end
+
+# Add logo for opensearch-dashboards
```
Comment (o-alex) [inline recipes/install.rb:894]: do we need this? does the rebranding work?

**Review by o-alex — CHANGES_REQUESTED** [review]

File: Berksfile (hunk 1 above)
Comment (jimdowling) [reply Berksfile:7]: Yep.

File: recipes/install.rb (hunk 3 above)
Comment (jimdowling) [reply recipes/install.rb]: removed

File: files/default/sql/ddl/updates/undo/2.4.0__undo.sql (hunk 2 above)
Comment (jimdowling) [reply files/default/sql/ddl/updates/undo/2.4.0__undo.sql]: had removed just before you saw this.

**Review by o-alex — APPROVED** [review]

**Pull request merged**